packaging/windows/Jenkinsfile.groovy

```diff
@@ -0,0 +1,72 @@
+    step([$class: 'S3BucketPublisher', dontWaitForConcurrentBuildCompletion: false, entries: [[bucket: 'prerelease.keybase.io', excludedFile: '', flatten: true, gzipFiles: false, keepForever: false, managedArtifacts: false, noUploadOnFailure: true, selectedRegion: 'us-east-1', showDirectlyInBrowser: false, sourceFile: 'src\\github.com\\keybase\\client\\packaging\\windows\\${BUILD_TAG}\\update-windows-prod-test-v2.json', storageClass: 'STANDARD', uploadFromSlave: true, useServerSideEncryption: false]], profileName: 'keybase', userMetadata: []])
+  }
+
+  if (UpdateChannel == "Smoke"){
+    def clientCommit = getCommit('src\\github.com\\keybase\\client')
+    def kbfsCommit =  getCommit('src\\github.com\\keybase\\kbfs')
+    def updaterCommit =  getCommit('src\\github.com\\keybase\\go-updater')
+    def releaseCommit =  getCommit('src\\github.com\\keybase\\release')
+    def smokeASemVer = ''
+    dir('src\\github.com\\keybase\\client\\go\\keybase') {
+        smokeASemVer = bat(returnStdout: true, script: '@echo off && winresource.exe -cv').trim()
+    }
+    build job: 'windows-installer-pipeline', parameters: [string(name: 'ClientRevision', value: "${clientCommit}"), string(name: 'KBFSRevision', value: "${kbfsCommit}"), string(name: 'UpdaterRevision', value: "${updaterCommit}"), string(name: 'ReleaseRevision', value: "${releaseCommit}"), string(name: 'DOKAN_PATH', value: "${DOKAN_PATH}"), string(name: 'UpdateChannel', value: 'Smoke2'), string(name: 'SmokeASemVer', value: "${smokeASemVer}")], wait: false
+  }
+  if (UpdateChannel == "Smoke2"){
+    stage 'Publish smoke updater jsons to S3'
+    step([$class: 'S3BucketPublisher', dontWaitForConcurrentBuildCompletion: false, entries: [[bucket: 'prerelease.keybase.io/windows-support', excludedFile: 'src\\github.com\\keybase\\client\\packaging\\windows\\${BUILD_TAG}\\update-windows-prod-test-v2.json', flatten: true, gzipFiles: false, keepForever: false, managedArtifacts: false, noUploadOnFailure: true, selectedRegion: 'us-east-1', showDirectlyInBrowser: false, sourceFile: 'src\\github.com\\keybase\\client\\packaging\\windows\\${BUILD_TAG}\\*.json', storageClass: 'STANDARD', uploadFromSlave: true, useServerSideEncryption: false]], profileName: 'keybase', userMetadata: []])
+    def smokeBSemVer = ''
+    dir('src\\github.com\\keybase\\client\\go\\keybase') {
+        smokeBSemVer = bat(returnStdout: true, script: '@echo off && winresource.exe -cv').trim()
+    }
+    withCredentials([[$class: 'StringBinding', credentialsId: 'KEYBASE_TOKEN', variable: 'KEYBASE_TOKEN']]) {
+      dir('src\\github.com\\keybase\\release') {
+          bat 'release announce-build --build-a="${SmokeASemVer}" --build-b="${smokeBSemVer}" --platform="windows"'
```

Comment:
I may need some help determining if this is working correctly - see the bottom of https://ci.keyba.se/job/windows-installer-pipeline/466/console